subprojects/docs/src/docs/userguide/java.adoc

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-[[java]]
+[[building_java_projects]]
```

Comment:
We will need to rename this file to buildingJavaProjects.adoc to preserve quick edit functionality